build/integrationtests/Dockerfile

```diff
@@ -7,15 +7,21 @@ LABEL maintainer Rob Bovill <rbovill@lsst.org>
 WORKDIR ${HOME}
 
 ARG integrationtests
+ARG robot_efd
 
 RUN source /home/saluser/.setup_sal_env.sh && \
 mamba install -c lsstts ts-integrationtests=${integrationtests} -y && \
 mamba install -c conda-forge robotframework && \
-pip3 install -U lsst-efd-client
+pip3 install -U PyYAML lsst-efd-client
 
 RUN git clone https://github.com/lsst-ts/robotframework_EFD.git
 
-ENV RUN_ARG="echo 'Do something here'"
+WORKDIR /home/saluser/robotframework_EFD
+
+RUN /home/saluser/.checkout_repo.sh ${robot_efd}
+
+ENV PYTHONPATH "$PYTHONPATH:/home/saluser/robotframework_EFD"
```

Comment:
Any chance `robotframework_EFD` might become a conda package one day? Then you don't have to be messing with `PYTHONPATH`.